tests/e2e/specs/wcpay/shopper/shopper-pay-for-order.spec.js

```diff
@@ -14,7 +10,6 @@ const { shopperWCP } = require( '../../../utils' );
 describe( 'Shopper > Pay for Order', () => {
 	beforeAll( async () => {
 		await shopper.login();
-		await createSimpleProduct();
```

Comment:
This would pollute the long-lived Atomic site with countless "Beanie" products, whereas there already exists one by default (since the initial setup phase requires importing some products through a CSV file).
So I didn't see a point in keeping this, we'll just target the existing "Beanie" product.